packages/cms-lib/ignoreRules.js

```diff
@@ -3,7 +3,10 @@ const path = require('path');
 const ignore = require('ignore');
 const findup = require('findup-sync');
 
-const ignoreRules = ignore().add('node_modules');
+const hiddenFilesRegEx = /(^|\/)\.[^/.]/g;
```

Comment:
I am pretty sure that this could be the string '.*' but might be worth adding a test or two to confirm.